### ISSUE TITLE
Question: When set pagination show to false and not using perPage, why is the table limiting to 10 rows?

### ISSUE THREAD

**Issue by samuelmoser** [matfish2/vue-tables-2#867]
- Vue.js version:
- consumed using: (browserify/webpack/rollup)
- FULL error message (including stack trace):
- relevant code:
- steps for reproducing the issue:

**Comment by matfish2** [comment]
show:false does just that, i.e hides the pagination. You can set perPage to a high value if you want to display all rows in one page.

**Comment by samuelmoser** [comment]
yeah we did that, but setPage: 50000 looks like a hack :-)

**Comment by jawa-the-hutt** [comment]
@samuelmoser not sure if this is still a concern of yours, but thought i'd let you and others know what I did to workaround this so that it doesn't show something like 50,000 in the drop down.

In my `perPageValues`, I provide an `All` option like this:  `perPageValues: [1, 5, 'All']`;

Then, I've used a custom template as described in the docs and adjusted the template for  `VtPerPageSelector.vue`.  All I've done here is adjust the options html to this:

```html
<option v-for="val in props.perPageValues" :value="val === 'All' ? 50000 : val">
  {{ val }}
</option>
```

Basically, if it sees the `All` value chosen it will then load a max of 50000 records.
